CHANGELOG.next.asciidoc

```diff
@@ -277,6 +277,7 @@ otherwise no tag is added. {issue}42208[42208] {pull}42403[42403]
 - Use namespace for GetListMetrics when exists in AWS {pull}41022[41022]
 - Only fetch cluster-level index stats summary {issue}36019[36019] {pull}42901[42901]
 - Upgrade `go.mongodb.org/mongo-driver` from `v1.14.0` to `v1.17.4` to fix connection leaks in MongoDB module {pull}44769[44769]
+- Increase default period from 10s to 60s {pull}44781[44781]
```

Comment:
Can we put this CL in "=== Added" section; it is the place where we put enhancement and new additions. I think the PR you created last or couple of weeks back, the CL entry was incorrect there as well. Can you check?

Reply:
There are ongoing discussions on where to put the changelog. Please see that too. 
[Reference](https://github.com/elastic/beats/pull/44576#discussion_r2137074745).